variables.tf

```diff
@@ -1,25 +1,91 @@
   default     = ""
 }
 
+variable "profile" {
+  description = "The profile name as set in the shared credentials file. If not set, it will be sourced from the ALICLOUD_PROFILE environment variable."
+  type        = string
+  default     = ""
+}
+variable "shared_credentials_file" {
+  description = "This is the path to the shared credentials file. If this is not set and a profile is specified, $HOME/.aliyun/config.json will be used."
+  type        = string
+  default     = ""
+}
+
+variable "skip_region_validation" {
+  description = "Skip static validation of region ID. Used by users of alternative AlibabaCloud-like APIs or users w/ access to regions that are not public (yet)."
+  type        = bool
+  default     = false
+}
+
+#####################
+# Cdn
+#####################
+variable "create_cdn_domain" {
+  description = "Whether to create a Cdn domain. Default to true."
+  type        = bool
+  default     = true
+}
+
 variable "domain_name" {
-  default = "jiaion.alicdn.com"
+  description = "Name of the accelerated domain. This name without suffix can have a string of 1 to 63 characters, must contain only alphanumeric characters or '-', and must not begin or end with '-', and '-' must not in the 3th and 4th character positions at the same time. Suffix '.sh' and '.tel' are not supported."
+  type        = string
+  default     = ""
 }
 
-variable "type" {
-  default = "web"
+variable "cdn_type" {
+  description = "Cdn type of the accelerated domain. Valid values are 'web', 'download', 'video'."
+  type        = string
+  default     = "web"
 }
 
-variable "origin_type" {
-  default = "domain"
+variable "sources" {
+  description = "The source address list of the accelerated domain. Each item can contains keys:'content'(The adress of source. Valid values can be ip or doaminName.),'type'(The type of the source. Valid values are 'ipaddr', 'domain' and 'oss'. Default to 'ipaddr'),'port'(The port of source. Valid values are '443' and '80'. Default value is '80'.),'priority'(Priority of the source. Valid values are '0' and '100'. Default value is '20'.),'weight'(Weight of the source. Valid values are from '0' to '100'. Default value is '10', but if type is 'ipaddr', the value can only be '10'.)."
+  type        = list(map(string))
+  default     = []
 }
 
-variable "origins" {
-  type = list(string)
+variable "scope" {
+  description = "Scope of the accelerated domain. Valid values are 'domestic', 'overseas', global'. Default value is 'domestic'. This parameter's setting is valid Only for the international users and domestic L3 and above users ."
+  type        = string
+  default     = "domestic"
+}
+
+variable "tags" {
+  description = "A mapping of tags to assign to the cdn."
+  type        = map(string)
+  default     = {}
+}
 
-  default = [
-    "jiaion.github.io",
-  ]
+#####################
+# Cdn_config
+#####################
+variable "add_config_domain_name" {
+  description = "List of cdn domain names for bluk configuration."
```

Comment:
List of added cdn domain names for batch configuration. If set, the 'set_config' should be true.